packages/vite/src/node/plugins/esbuild.ts

```diff
@@ -306,7 +306,9 @@ export function resolveEsbuildTranspileOptions(
   // https://github.com/vuejs/core/issues/2860#issuecomment-926882793
   const isEsLibBuild = config.build.lib && format === 'es'
   const esbuildOptions = config.esbuild || {}
+
   const options: TransformOptions = {
+    charset: 'utf8',
```

Comment:
Should we do the same for [dev](https://github.com/vitejs/vite/blob/d5fe92cd2a0be2b8636e876a81a63921a808afb2/packages/vite/src/node/plugins/esbuild.ts#L181-L193) and the [optimizer](https://github.com/vitejs/vite/blob/d5fe92cd2a0be2b8636e876a81a63921a808afb2/packages/vite/src/node/optimizer/index.ts#L589-L597) too?

Reply:
Good point, yes, @azat-io could you also set the default to UTF8 in these esbuild configs?